apps/web/src/server/applications/create/applicant/applications-create-applicant.service.js

```diff
@@ -83,3 +83,22 @@ export async function getNthApplicant(applicationId, applicantNumber) {
 
 	return applicants && applicants.length > 0 ? applicants[applicantNumber] : null;
 }
+
+/**
+ * Returns the applicant matching id of a draft application
+ *
+ * @param {number} applicationId
+ * @param {number} applicantId
+ * @param {null | string} query
+ * @returns {Promise<Applicant|null>}
+ */
+export async function getApplicantById(applicationId, applicantId, query = null) {
+	const { applicants } = await getApplicationDraft(applicationId, query);
```

Comment:
can't we say that `query` here is always 'applicants:true'?

Reply:
it is different for the address call